ErrorBlocks.md

```diff
@@ -0,0 +1,156 @@
+  4. Create the graveyard volume (void optional for understanding).
+  5. Finish part in Cubit/Trelis.
+     * Imprint and Merge volumes
+     * Assign materials
+     * Export file to .h5m
+
+#### Goals of Example
+
+  * Execute the workflow with a model that has geometric issues to be resolved in SpaceClaim.
+  * Work through important processes on an example that is easy to visualize and understand.
+  * Provide a general understanding of what issues could arise on more complex models.
+
+## Procedure
+
+### In Initial CAD program
+
+_Any CAD engine should work for this process as long as it can export to a format importable by SpaceClaim or Cubit/Trelis._
+
+
+  1. Download the part titled "ErrorBlocks.sw" from this page and open in Solidworks.
+  2. Export the assembly/part to ACIS format (.sat file extension)
+     * ACIS format 22 is preferred (default for Solidworks)
+     * Use save or export feature and select ACIS (.sat) format
+
+![ErrorBlocksOverviewWide](https://github.com/NybergWISC/model_development_workflow/blob/master/PicturesSnapshots/ErrorBlocks/ErrorBlocksOverviewWide.JPG)
+
+<details><summary> Notes </summary><p>
+- The preferred format is ACIS (.sat). Note that multiple file types could be used, but there is anecdotal evidence that suggests .sat files work the best for this process. Otherwise formats such as STEP and STL are also able to be imported to Trelis/Cubit.
+<br/>
+- This step is unnecessary unless you want to explore the model in Solidworks or learn how to save file as .sat.
+<br/>
+- To skip this step download the ACIS file titled "ErrorBlocks.sat" and move onto ANSYS SpaceClaim section.
+</p>
+</details>
+
+### In ANSYS SpaceClaim
+_SpaceClaim is useful for cleaning models that have basic issues such as small gaps, overlaps, edge contacts, and unnecessary geometric detail. More general SpaceClaim tutorials/support can be found_ [here](http://www.spaceclaim.com/en/Support/Tutorials/Essentials.aspx).
+  1. Open ErrorBlocks.sat by clicking file, open, and then sorting by .sat type files.
+  2. Identify all basic issues which exist within the model
+
+      * Visually:
+        - Find one example of each basic issue as described below. All four are relatively conspicuous and easy to find visually.
+      * With SpaceClaim tools:
+        - To identify gaps navigate from __Prepare -> Clearance -> Detect__. In this case to find the gap enter a maximum distance of `1cm` in the Find Options box. The gap will be highlighted in red.
+        - To identify interference navigate from  __Prepare -> Interference -> Remove.__ There are technically two volumes of interference and both will be highlighted in red.
+        - The __Imprint__ tool highlights any coincident edges or faces within the part and can make it easier to find edge contacts as well as overlaps. This tool can be found by going __Prepare -> Analysis -> Imprint__.
+        - There are no built-in tools in SpaceClaim to generally identify unnecessary details.

```

Comment:
My thought was that this would be a little too intimidating and it would be easier to follow if those sections were broken up. Also markdown shrinks the image a lot when there are a large number of columns which lead me to use less.

Do you think it would be better to do a separate table for this section with columns: Type of Issue, Description of Issue, SpaceClaim tool (possibly another picture column as well)?

Reply:
I think the more important piece here is that you should introduce an example of what the issues are (description with picture) before you introduce how to detect it and fix it. Right now you kind of have it flipped. Adding it to the table is one way I see it being fixed, but you could do it a different way if you think the formatting of the table would be too difficult.